src/it/scala/temple/generate/language/service/GoGeneratorIntegrationTest.scala

```diff
@@ -0,0 +1,29 @@
+package temple.generate.language.service
+
+import org.scalatest.{BeforeAndAfter, Matchers}
+import temple.containers.GolangSpec
+import temple.generate.docker.ast.DockerfileRoot
+import temple.generate.docker.ast.Statement._
+
+class GoGeneratorIntegrationTest extends GolangSpec with Matchers with BeforeAndAfter {
+
+  behavior of "GolangValidator"
```

Comment:
\n

Reply:
We haven't been very consistent with this... See:

https://github.com/TempleEight/temple/blob/07b33ef2cb0023fe2bcf3019b6b2289f1e3071c4/src/it/scala/temple/generate/database/PostgresGeneratorIntegrationTest.scala#L24-L25

https://github.com/TempleEight/temple/blob/07b33ef2cb0023fe2bcf3019b6b2289f1e3071c4/src/test/scala/temple/generate/database/PostgresGeneratorTest.scala#L10-L12

Which do we prefer? I'll fix them all in a follow up.

Reply:
I like the newline personally